armi/reactor/blocks.py

```diff
@@ -757,8 +757,11 @@ def adjustDensity(self, frac, adjustList, returnMass=False):
         if returnMass:
             # do this with a flag to enable faster operation when mass is not needed.
             volume = self.getVolume()
-        for nuclideName in adjustList:
-            dens = self.getNumberDensity(nuclideName)
+
+        numDensities = self.getNuclideNumberDensities(adjustList)
+
+        for nuclideName,dens in zip(adjustList,numDensities):
```

Comment:
Lack of space after `,` indicates that you should run `black` on this file to get formatting perfect.

Reply:
Right, forgot about that.  Should be done.